config/params/mainnet_config.go

```diff
@@ -110,11 +110,13 @@ var mainnetBeaconConfig = &BeaconChainConfig{
 	Eth1FollowDistance:               2048,
 
 	// Fork choice algorithm constants.
-	ProposerScoreBoost:              40,
+	ProposerScoreBoost:              20,
 	ReorgWeightThreshold:            20,
 	ReorgParentWeightThreshold:      160,
 	ReorgMaxEpochsSinceFinalization: 2,
-	IntervalsPerSlot:                3,
+	IntervalsPerSlot:                4,
```

Comment:
This cannot change globally, you need to add a new `IntervalsPerSlotEPBS` or you will break previous forks.

Reply:
updated